cmd/argo/commands/server.go

```diff
@@ -104,7 +109,7 @@ See %s`, help.ArgoSever),
 		defaultBaseHRef = "/"
 	}
 	command.Flags().StringVar(&baseHRef, "basehref", defaultBaseHRef, "Value for base href in index.html. Used if the server is running behind reverse proxy under subpath different from /. Defaults to the environment variable BASE_HREF.")
-	command.Flags().StringVar(&authMode, "auth-mode", "server", "API server authentication mode. One of: client|server|hybrid")
+	command.Flags().StringArrayVar(&authModes, "auth-mode", []string{"server"}, "API server authentication mode. One of: client|server|hybrid")
```

Comment:
Why is this changed to be repeatable? What is the behavior of `--auth-mode server --auth-mode hybrid` ?

Reply:
I see it was to add `sso` as an option. I feel like the presence of SSO configuration should presume that option is available.